app/cells/plugins/core/asset_info/index.haml

```diff
@@ -1,2 +1,2 @@
 - if asset
-  = image_tag(asset['style_urls'][config[:thumbnail_style]], height: '50px')
+  = image_tag(asset_thumb(asset), height: '50px')
```

Comment:
@MKwenhua where did this method come from? It doesn't exist in my checked out copy, so I'm nixing it for now.